PaddleCV/image_classification/scripts/train/ResNet50_dist.sh

```diff
@@ -0,0 +1,19 @@
+##Training details
+#GPU: NVIDIA® Tesla® V100 4cards 120epochs 67h
```

Comment:
update the training info?

Reply:
have not tested this completely, just removed.